zio-http/src/main/scala/zhttp/service/server/content/compression/CompressionOptions.scala

```diff
@@ -0,0 +1,40 @@
+package zhttp.service.server.content.compression
+
+import io.netty.handler.codec.compression.{CompressionOptions => JCompressionOptions, StandardCompressionOptions}
+import zhttp.service.server.content.compression.CompressionOptions.Kind
+
+final case class CompressionOptions(level: Int, bits: Int, mem: Int, kind: Kind) { self =>
+  def toJava: JCompressionOptions = self.kind match {
+    case CompressionOptions.GZip    => StandardCompressionOptions.gzip(self.level, self.bits, self.mem)
+    case CompressionOptions.Deflate => StandardCompressionOptions.deflate(self.level, self.bits, self.mem)
+  }
+
+}
+
+object CompressionOptions {
+  sealed trait Kind
```

Comment:
maybe `CompressionType` might be a better name

Reply:
The motivation behind keeping it `CompressionOptions` is to keep it relatable with netty's CompressionOptions.